docs/docusaurus.config.js

```diff
@@ -0,0 +1,121 @@
+// @ts-check
+// Note: type annotations allow type checking and IDEs autocompletion
```

Comment:
do you think it would be useful to add basic comments about each section?

Reply:
I personally think it's not necessary to add lots of inline comments because the docusaurus docs are really helpful (I can add a link to the docs) but I'm happy to add additional inline docs especially if there are areas/sections you feel are unclear.